apps/api/e2e/setup.ts

```diff
@@ -1,31 +1,27 @@
 /**
- * Setup script for end-to-end tests
+ * Global Setup
  *
- * This file is automatically run by Jest before all tests.
- * It initializes the test environment, including the database and server.
+ *
+ * Initializes the test environment, including the database and server.
  */
 import { config } from "dotenv";
 import fs from "fs";
-import { Server } from "http";
 import path from "path";
 
-import { dbManager } from "./utils/db-manager.js";
-import {
-  killExistingServers,
-  startServer,
-  stopServer,
-} from "./utils/server.js";
+import { createLogger } from "@/lib/logger.js";
 
-// Store global server reference
-let server: Server;
```

Comment:
Here's an example of a place where the Server instance is not that actual server under test, and really doesn't have any purpose.